websites/B/BoxSeries/presence.ts

```diff
@@ -0,0 +1,162 @@
+const presence = new Presence({
+		clientId: "955350055034945576"
+	}),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+		browsing: "presence.activity.browsing"
```

Comment:
These strings will be removed soon, use `general.` instead. You can find the strings used in [PreMiD/Localization](https://github.com/PreMiD/Localization/blob/main/src/Presence/general.json)